02-Login-With-Profile/README.md

```diff
@@ -0,0 +1,72 @@
+# Passage Example Angular App
+
+This example application uses the Passage Element in a Angular application to authenticate users using biometrics or magic links. The server uses the 
```

Comment:
Mention the profile in this overview too

Reply:
added a mention of profile here (and removed talking about passage-node & the old backend folder), let me know what you think of this and I'll add it to react too